lib/internal/cluster/master.js

```diff
@@ -12,6 +12,18 @@ const cluster = new EventEmitter();
 const intercom = new EventEmitter();
 const SCHED_NONE = 1;
 const SCHED_RR = 2;
+var debugPortOffset = 0;
+
+const debugArgRegex = /^(--inspect|--debug|--debug-(?:brk|port))/;
```

Comment:
Nit: I think this is identical to this RegEx:  `/^(--inspect|--debug)/`.  Not sure if you want it lenient like that or if you meant to have it be more strict by including `$/` at the end and accomodating `=1234` type stuff.